index.js

```diff
@@ -1,11 +1,2 @@
 #!/usr/bin/env node
-const AddDependencies = require('./lib/AddDependencies');
-
-console.log('\x1b[33m%s\x1b[0m', 'This script adds dependencies (latest or specified versions) to the package.json file skipping the installation process.');
-
-const app = new AddDependencies();
-
-app.addDependencies().then(app.saveToPackage.bind(app)).catch((error) => {
```

Comment:
this was one change i was unsure of, as to why `app.saveToPackage.bind(app)` was being done?

Reply:
Obviously, because `saveToPackage` method is supposed to use the class context.

Reply:
but why does it not have the class context from within itself already?